commons-test/src/main/java/org/infinispan/commons/test/CustomJUnitXMLWriter.java

```diff
@@ -9,15 +9,15 @@
 import javax.xml.stream.XMLStreamWriter;
 
 /**
- * A JUnit XML report generator for Polarion based on the JUnitXMLReporter
+ * A JUnit XML report to prevent crashing the surefire fork JVM based on JUnitXMLReporter.
  *
- * <p>Extracted from {@link PolarionJUnitXMLReporter}</p>
+ * This class is responsible to log hanged tests and leaked threads.
```

Comment:
```suggestion
 * Based on JUnitXMLReporter.
```
Not really, it's responsible for reporting in the JUnit XML report format whatever other classes decide to report. For now, it's just leaked threads.